intro/webhooks_intro.md

```diff
@@ -193,27 +193,30 @@ The *Status* of the registration should show as **Active**. If the registration
 
 ![Event Registration Details tab in Adobe Developer Console](../img/events-registration-details.png)
 
-### Troubleshooting a Disabled Registration Status
+### Troubleshooting Unstable/Disabled Registration Status
 
-If you made an error transcribing the webhook URL, Adobe Events&rsquo; test of your webhook would have failed, resulting a **Disabled** status.
+If you made an error transcribing the webhook URL, Adobe I/O Events' test of your webhook would have failed, resulting in a **Disabled** status.
 
-In general, Adobe I/O Events will always confirm that your webhook received an event by means of the response code your webhook sends to each HTTP POST request. 
+In general, `Adobe I/O Events` will always confirm that your webhook received an event by means of the response code your webhook sends to each HTTP POST request. 
 
-If Adobe fails to receive a 200 OK response code within 10 seconds, it retries the request, including a special header: `x-adobe-retry-count`. 
-The value of this header begins at 1. 
-If the first retry request fails as well, Adobe waits, then retries again, incrementing the value of `x-adobe-retry-count` with each retry until it reaches 5. 
-Each wait interval is the square of the previous interval. 
-Once five retries are attempted (after 31 minutes) and the last attempt also fails, 
-Adobe marks the webhook as invalid and stops sending requests. 
+If `Adobe I/O Events` fails to receive a successful response code from your webhook within 10 seconds, it retries the request, including a special header `x-adobe-retry-count` (this header indicates how many times the delivery of an event or a batch of events has been attempted).
 
-To restart the flow of requests, once you have fixed the problem preventing your webhook from responding, 
-you must log into the `Adobe Developer Console`, edit your events registration, 
-it will re-trigger a webhook challenge request, and eventually a webhook re-activation.
+`Adobe I/O Events` will keep on retrying delivery to your webhook for **24 hours** using exponential and fixed backoff strategies. The first retry is attempted after 1 minute and the period between retries doubles after each attempt (second retry is after 2m, etc.), but is at most 15 minutes.
+
+If an event isn't delivered after 2 hours of retries, `Adobe I/O Events` marks the webhook as **Unstable**, but still keeps on attempting delivery. This gives sufficient time to fix whatever problems your are facing and avoid your webhook from getting marked as Disabled.
+
+If all retry attempts get exhausted and the event still isn't delivered (webhook not responding or responding with a non `2XX` response), `Adobe I/O Events` drops the events, marks the webhook as **Disabled**, and stops sending any further events.
```

Comment:
"marks the webhook as **Disabled**" OR "marks the event-registration as **Disabled**"? I notice that we are using 'webhook' instead of 'event-registration'. Is it to remain consistent with the older version of this doc?

Reply:
Here we are using "event-registration" - https://github.com/AdobeDocs/adobeio-events/pull/87/commits/a0d1856e68b8ddba5f3f4300811836280683c8db#diff-7ba92183051d83f15ba331c96cdaa686e48ec16f9e97c46a30f3789027c3dfdbR96

Reply:
Hmm good point. Shall we change this throughout with `webhook-registration`?

Reply:
On the Developer Console, we use `event-registration`. I think we should remain consistent and use that only.

Reply:
My only concern is that throughout this doc, we have been using `webhook` heavily. Changing it to `webhook-registration` would be a less transparent change and still convey what we are trying to say.

Reply:
Introducing a new term `webhook-registration` is not preferable. Either we continue to use `webhook` (and leave this inconsistency to be handled in the future), or we become consistent by using `event registration` everywhere.

Reply:
After discussion with @sgera, `webhook-registration` will be yet another new term, so it would be good if we use `event-registration` throughout.

I'll make those changes.

Reply:
Changed to `event registration` throughout based on the below console screenshot:

![event-registration](https://user-images.githubusercontent.com/6910192/141435245-a977188d-7a55-450d-8db9-7223d289d764.png)

Reply:
Looks really good, thank you!

Reply:
The [SDK webhook page](https://www.adobe.io/apis/experienceplatform/events/docs.html#!adobedocs/adobeio-events/master/sdk/sdk_webhooks.md) is full of `webhook registration` terms, so we can't get rid of it. It's there in our SDK methods as well. However, I have updated the intro section to align with the terminology. See https://github.com/AdobeDocs/adobeio-events/pull/87/commits/8f1c0147ef7215e91d0a2bc6694790589312ab48.